DashKitUI/View Models/DashSettingsViewModel.swift

```diff
@@ -218,9 +218,9 @@ extension PumpManagerStatus.BasalDeliveryState {
             let rate = dose.unitsPerHour - scheduledRate
             
             if rate < 0 {
-                return rate / dose.unitsPerHour
+                return rate / scheduledRate
             } else {
-                return rate / (maximumTempBasalRate - dose.unitsPerHour )
+                return rate / (maximumTempBasalRate - scheduledRate )
```

Comment:
I'm not sure I agree with this calculation. What does `maximumTempBasalRate` have to do with the `netBasalPercent`?

Let's say it is a low-temp situation where the current scheduled rate is `1.0` U/hr and the current temp basal rate is `0.5` U/hr.  Thus:
```
  rate = 0.5 - 1.0 = -0.5
  netBasalPercent = -0.5 / 1.0 = -0.5 // e.g. -50% - makes sense
```
Now, let's say it is a high-temp situation where the current scheduled rate is `1.0` U/hr and the current temp basal rate is `1.5` U/hr. For our first example, let's say the maximum temp basal rate is `3.0` U/hr.
```
  rate = 1.5 - 1.0 = 0.5
  netBasalPercent = 0.5 / (3.0 - 1.0) = 0.25 // eg. 25% - hmm
```
Let's change the the maximum temp basal rate to `5.0` U/hr, but leave everything else the same.
```
  rate = 1.5 - 1.0 = 0.5
  netBasalPercent = 0.5 / (5.0 - 1.0) = 0.125 // eg. 12.5% - what?
```
Why would the `netBasalPercent` differ just because the `maximumTempBasalRate` was different? It is still the same 50% increase above the scheduled rate.

It seems to me that the correct calculation is just (regardless of low- or high-temp):
```
return rate / scheduledRate
```

But perhaps I'm just totally confused here. Or perhaps it has always been this way. 🤷‍♂️

Reply:
[nit] Also, extra space before the `)`.

Reply:
The temp basal widget, as has always been implemented in the Loop HUD, and now in pod settings, can go from +100% to -100%. The top is max basal, the bottom is 0U/hr, and the middle is scheduled basal. The top half will only be proportional when max basal rate is 2x your scheduled basal rate. Otherwise, it's not proportional. Does that answer your question?

Reply:
Yes, it does. Never knew that and not intuitive for me at least. My default expectation that the % was always based upon your scheduled basal rate.

Reply:
Yeah, I can see why you might expect that. The graphical portion of the temp basal widget is designed to give a rough relative sense of the action Loop is taking, not a precise value.